test.sh

```diff
@@ -8,6 +8,19 @@
 
 set -e
 
+if [ -n "${TEST_OPENSHIFT_4}" ]; then
+  export OS=${OS:-rhel7}
+  if [ "${OS}" == "rhel7" ] ; then
+    NAMESPACE=${NAMESPACE:-rhscl/}
+    REGISTRY=${REGISTRY:-registry.access.redhat.com/}
+  else
+    NAMESPACE=${NAMESPACE:-centos/}
+  fi
+
+  export NAMESPACE
+  export REGISTRY
+fi
+
```

Comment:
Is this part needed? I do not see a mention of `$REGISTRY` or `$NAMESPACE` in the nodejs PR. OS should always be set at this point (from Makefile).